src/components/movieDetails/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import MovieGenres from '../movieGenres';
+import useStyles from './styles';
+import { IMovieDetailsProps } from './types';
+
+const MovieDetails: React.FC<IMovieDetailsProps> = ({
+  movieName,
+  releaseYear,
+  imageUrl,
+  rating,
+  genres,
+  duration,
+  description,
+}) => {
+  const styles = useStyles();
+
+  const durationToHM = (totalMinutes: number) => {
```

Comment:
It's better to extract such helpers/utils to a separate file. It is pure, and might be tested in isolation.

Reply:
moved to utils